src/components/MessageBox/MessageBox.tsx

```diff
@@ -6,17 +6,32 @@ type Props = {
   children: ReactNode;
   incoming?: boolean;
   classNames?: string;
+  translation?: string;
+  showTranslation?: boolean;
```

Comment:
Boolean props are a code smell. If the translation prop if provided, we should show it. No need for this redundant prop. Also please update `.github/agents/Codermans/agent.md` with a note to this effect. It can be in a new section of that document labelled `## Coding Style Tips`